src/pages/keyblocks/[id].vue

```diff
@@ -14,7 +14,7 @@
     v-if="keyblockDetails"
     :keyblock-details="keyblockDetails"/>
 
-  <app-tabs v-if="!keyblockDetails.notExistent">
+  <app-tabs v-if="keyblockDetails && !keyblockDetails?.notExistent">
```

Comment:
fix for error showing up when navigating to other keyblock details pages using the navigation buttons next to keyblock height

Reply:
I would prefer to have this in separate computed variable as that what Vue styleguide suggests